src/tests/unit/tests/common/telemetry-data-factory.test.ts

```diff
@@ -319,6 +320,22 @@ describe('TelemetryDataFactoryTest', () => {
         expect(actual).toEqual(expected);
     });
 
+    test('forSelectGettingStarted', () => {
+        const event = mouseClickEvent;
+        const expected: SelectGettingStartedTelemetryData = {
+            triggeredBy: 'mouseclick',
+            source: TelemetryEventSource.DetailsView,
+            selectedTest: VisualizationType[VisualizationType.Headings],
```

Comment:
Might want to store `VisualizationType.Headings` as a variable that is referenced twice.